src/opendr/perception/object_tracking_2d/fair_mot/algorithm/lib/models/networks/dlav0.py

```diff
@@ -392,14 +392,17 @@ def forward(self, x):
 
             return x
 
-    def load_pretrained_model(self, data="imagenet", name="dla34", hash="ba72cf86"):
+    def load_pretrained_model(self, load=True, data="imagenet", name="dla34", hash="ba72cf86", num_classes=None):
```

Comment:
With the new changes, i think that this method might need to be renamed as it no longer just loads a pretrained model, but might just create a model based on its `load` argument which is provided with the value of `pretrained` from methods below.

Would it be better to name it `load_model` and the argument load renamed to `pretrained`?